bin/ansible

```diff
@@ -41,6 +41,15 @@ from ansible.errors import AnsibleError, AnsibleOptionsError, AnsibleParserError
 from ansible.module_utils._text import to_text
 
 
+# Used for determining if the system is running a new enough python version
+# and should only restrict on our documented minimum versions
+_PY3_MIN = sys.version_info[:2] >= (3, 5) and sys.version_info[0] == 3
+_PY2_MIN = sys.version_info[:2] >= (2, 6) and sys.version_info[0] == 2
+_PY_MIN = _PY3_MIN or _PY2_MIN
+if not _PY_MIN:
+    raise SystemExit('ERROR: Ansible requires a minimum of Python 2.6 or Python 3.5. Current version: %s' % ''.join(sys.version.splitlines()))
```

Comment:
I'd maybe write this as "[...]Python2 2.6 or Python3 3.5[...]" for people a little less familiar with the Python releases.